zqd/space/space.go

```diff
@@ -8,9 +8,12 @@ import (
 	"path/filepath"
 
 	"github.com/brimsec/zq/pkg/fs"
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zqd/api"
 )
 
 const configFile = "config.json"
+const infoFile = "info.json"
```

Comment:
I'm a little concerned about spreading space related information across files. What's your thoughts on using a single file (with a name other than 'config') to hold configuration & metadata?

Reply:
That's a fair point and I could go either way on that one - if you think this is preferable I can adjust the PR. That said, there's feedback below saying this stuff should go in an altogether different place, so I'll try to resolve that question offline first...

Reply:
Offline resolution was that info should move to the data dir, which would prevent colocating it with config.